ste/xfer-setProperties.go

```diff
@@ -193,7 +193,18 @@ func setPropertiesFile(jptm IJobPartTransferMgr) {
 		transferDone(common.ETransferStatus.Failed(), err)
 		return
 	}
-	srcFileClient := s.NewShareClient(jptm.Info().SrcContainer).NewRootDirectoryClient().NewFileClient(jptm.Info().SrcFilePath)
+
+	sourceShare := s.NewShareClient(jptm.Info().SrcContainer)
```

Comment:
is it possible to set properties or delete on a share snapshot file?

Reply:
Share snapshot can be deleted but not modified (https://learn.microsoft.com/en-us/azure/storage/files/storage-snapshots-files#capabilities). I will update this accordingly.

Reply:
that's a share snapshot, but what about a file in the share snapshot?

Reply:
yeah, I guess that's not possible (https://learn.microsoft.com/en-us/rest/api/storageservices/delete-file2#remarks).